lib/uniware/client.rb

```diff
@@ -142,6 +142,21 @@ def create_or_update_vendor_item(data, code)
       perform_operation("CreateOrEditVendorItemTypeRequest", body, facility_endpoint("#{code}"))
     end
 
+    def create_gatepass(data, code)
```

Comment:
The 'code' parameter should be called 'facility_code' across the board. Else scope for confusion with order code, item code etc.